src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1625,5 +1637,123 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_Microsoft_PowerShell_Ma
                     .EndEntry()
                 .EndList());
         }
+
+        private static IEnumerable<FormatViewDefinition> ViewsOf_Microsoft_PowerShell_Commands_TestConnectionCommand_PingStatus()
+        {
+            yield return new FormatViewDefinition(
+                "Microsoft.PowerShell.Commands.TestConnectionCommand+PingStatus",
+                TableControl.Create()
+                    .AddHeader(Alignment.Right, label: "Ping", width: 4)
+                    .AddHeader(Alignment.Left, label: "Source", width: 16)
+                    .AddHeader(Alignment.Left, label: "Address", width: 25)
+                    .AddHeader(Alignment.Right, label: "Latency(ms)", width: 7)
+                    .AddHeader(Alignment.Right, label: "BufferSize(B)", width: 10)
```

Comment:
It is the intention, yes. I find new PowerShell users tend to interpret header names as the literal property names, so including the units on the same row leads to confusion. Compromise offered by @SteveL-MSFT and co was to put units on the next line.

Reply:
If we think about a text search or copy-paste in Excel one line header looks more preferably.

Reply:
Why do you wonder if we have a request for Select-String to search in output text? :-)

Reply:
Because people like to search the plaintext from the formatter sometimes. I don't see how that correlates to Excel, though? 🙂 

Point taken about the text search though, I suppose, though I don't think it would be a huge obstacle in practice.

Reply:
Since this is formatting, we can make changes in the future based on user feedback